src/dune/docker.py

```diff
@@ -66,21 +70,58 @@ def get_container(self):
     def get_image(self):
         return self._image
 
-    def execute_docker_cmd(self, cmd):
-        with subprocess.Popen(['docker'] + cmd,
-                              stdout=subprocess.PIPE, stderr=subprocess.PIPE) as proc:
-            stdout, stderr = proc.communicate()
-            if self._cl_args.debug:
-                print('docker '+' '.join(cmd))
-                print(stdout.decode('UTF-8'))
-                print(stderr.decode('UTF-8'))
-        return [stdout.decode('UTF-8'), stderr.decode('UTF-8'), proc.poll()]
+    @staticmethod
+    def print_streams(stdout, stderr):
+        if stdout is None and stderr is None:
+            print('!! No stdout/stderr info captured...')
```

Comment:
Can you remove this line?
Alternatively, print it only in debug mode in form `print('No stdout/stderr captured')`